src/components/comparison/ComparisonForm.tsx

```
@@ -0,0 +1,93 @@
+                                 onChangeTopic,
+                                 onChangeCountry1,
+                                 onChangeCountry2,
+                                 onSubmit,
+                               }: ComparisonFormProps) => (
+  <Card className="mb-8 bg-white p-6">
+    <div className="space-y-4">
+      {/* 비교 주제 입력 */}
+      <div>
+        <label className="mb-2 block text-sm">비교 주제</label>
+        <Input
+          placeholder="예: 음주운전 처벌, 근로시간 제한, 마약 처벌 등"
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)